src/Extensions/SiteTreeContentReview.php

```diff
@@ -662,6 +665,8 @@ protected function setDefaultReviewDateForInherited()
 
         if (is_object($nextDate)) {
             $this->owner->NextReviewDate = $nextDate->getValue();
+        } elseif ($nextDate === false) {
+            $this->owner->NextReviewDate = null;
```

Comment:
How would it be false? You updated the return values of `getReviewDate()` to _not_ return false.

Reply:
`obj()` _cannot_ return false, assuming `$page` is a `ModelData` (which `SiteTree` obviously is and therefore I think this is meant to be). It's explicitly typehinted to `?object`